src/core/models.py

```diff
@@ -1793,7 +1793,7 @@ def __str__(self):
 
 
 class LoginAttempt(models.Model):
-    ip_address = models.GenericIPAddressField()
+    ip_address = models.GenericIPAddressField(blank=True, null=True)
```

Comment:
Is it even worth registering a login attempt if we do not have an IP address?
 
As this model is only used to count the number of failed attempts from a given IP, we might use the python logger as INFO for correlation but ignore the write to db

Reply:
yes please, and thanks